### PR TITLE
OBPIH-1328: Add import and export of picklist items

### PR DESCRIPTION
**Pull request by pmuchowski**

### CODE REVIEW

File: grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy [hunk 1]
```diff
@@ -161,6 +167,108 @@ class StockMovementApiController {
         forward(action: "read")
     }
 
+    def exportPickListItems = {
+        StockMovement stockMovement = stockMovementService.getStockMovement(params.id, "4")
+
+        List<PicklistItem> picklistItems = stockMovement?.pickPage?.pickPageItems?.inject([]) { result, pickPageItem ->
+            result.addAll(pickPageItem.picklistItems)
+            result
```
Comment (awalkowiak) [inline grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy]: Is this line required?

File: grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy [hunk 2]
```diff
@@ -161,6 +167,108 @@ class StockMovementApiController {
...
+                        binLocation: availableItem.binLocation,
+                        quantity: quantityPicked,
+                        sortOrder: pickPageItem.sortOrder
+                ))
+            }
+
+            stockMovementService.createOrUpdatePicklistItem(stockMovement)
+
+        } catch (Exception e) {
+            // FIXME The global error handler does not return JSON for multipart uploads
```
Comment (awalkowiak) [inline grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy]: Would this fix be very time consuming?

File: grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy (hunk 1 above)
Comment (pmuchowski) [reply grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy]: it's to return the result

File: grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy (hunk 2 above)
Comment (pmuchowski) [reply grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy]: it's copied from items import (on add items page) and I don't have any time left in the ticket

File: grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy (hunk 1 above)
Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy]: when I was looking on some example it was not necessary, but i guess this is not a bad approach

**Pull request merged**